curator/utils.py

```diff
@@ -1173,9 +1173,9 @@ def snapshot_running(client):
     :rtype: bool
     """
     try:
-        if settings.aws_flag:
+        # Check to see if we're running AWS Elasticsearch and if so, iterate through each repository to check for running snapshots
+        if settings.aws_versions['5.3.2'] == client.info()['version']:
```

Comment:
Maybe a loop here?

```python
is_aws = False
found_version = client.info()['version']
for aws_version in settings.aws_versions:
    if found_version == aws_version:
        is_aws = True
```
That way it will set the flag true if even one of the AWS versions matches.  It won't matter if the others don't.